utils/litemap/Cargo.toml

```diff
@@ -17,3 +17,7 @@ readme = "./README.md"
 
 [dependencies]
 serde = {version = "1.0.117", optional = true}
+
+[dev-dependencies]
+serde = "1.0.117"
+serde_json = "1.0.59"
```

Comment:
I noticed that you put absolute version rather than more generic. If I understand correctly the only consequence is that if someone has lower version of serde (say `1.0.104`) and vendors in `litemap` they'll bump to `1.0.117` automatically.

Is there a reason you don't just set `serde = "1"`?

Reply:
It's because I use `cargo add` and then edit the version when I realize it's already in the deptree.

Reply:
I fixed it